modules/gapi/include/opencv2/gapi/cpu/stereo.hpp

```diff
@@ -0,0 +1,25 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_CPU_STR_API_HPP
+#define OPENCV_GAPI_CPU_STR_API_HPP
+
+#include <opencv2/gapi/gkernel.hpp> // GKernelPackage
+
+namespace cv {
+namespace gapi {
+namespace stereo {
+namespace cpu {
+
+GAPI_EXPORTS_W GKernelPackage kernels();
```

Comment:
I believe it shouldn't be in python now, `GAPI_EXPORTS` would be enough

Reply:
done